server/server.py

```diff
@@ -1,6 +1,6 @@
 from flask import Flask, request, jsonify
 from flask_cors import CORS
-from server.nn import NN
+from nn import NN
```

Comment:
Good call! I was about to deprecate the server.py, or edit it so that it plays the whole tournament server side and returns the results as JSON. But until then, that should have been an unqualified import :sweat_smile: